test/test_nn.py

```diff
@@ -2068,9 +2069,10 @@ def test_bilinear(self):
         output3.backward(grad)
         gi1 = input1_1.grad.data.clone()
         gi2 = input2_1.grad.data.clone()
+        grad_input2 = module2.backward([input1, input2], grad)
```

Comment:
This seems like a hack and can break any time. Can you just remove the assert for now? I'll open an issue and someone will fix the test, you don't have to do it in this PR.